client/flags/flags.go

```diff
@@ -112,6 +112,7 @@ func AddTxFlagsToCmd(cmd *cobra.Command) {
 
 // AddPaginationFlagsToCmd adds common pagination flags to cmd
 func AddPaginationFlagsToCmd(cmd *cobra.Command, query string) {
+	cmd.Flags().Uint64(FlagPage, 0, fmt.Sprintf("pagination page of %s to query for", query))
```

Comment:
What do you think about adding a deprecation message here too? "this flag is being deprecated, prefer to use --offset"